File: 2007-Pycon-Paris/ex_exception/exemple.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

#define TAILLE 10

int main()
{
    char *tampon;
    size_t len;

    tampon = (char *)malloc(TAILLE);

    printf("Entrez votre nom : ");
    fflush(stdout);
    fgets(tampon, TAILLE, stdin);

    len = strlen(tampon);
    if (tampon[len-1] == '\n') {
        tampon[len-1] = '\0';
    }

    printf("Bonjour %s !\n", tampon);
    exit(EXIT_SUCCESS);
}

File: 2007-Pycon-Paris/ex_exception/exemple.py
from sys import exit

def main():
    tampon = raw_input("Entrez votre nom : ")

    if tampon[len(tampon)-1] == '\n':
        tampon = tampon[:len(tampon)-1]

    print "Bonjour %s !" % tampon
    exit(0)

if __name__ == "__main__":
    main()

